qiskit/providers/aer/openpulse/qobj/digest.py

```diff
@@ -199,6 +176,8 @@ def digest_pulse_obj(qobj):
 
     out.global_data['freqs'] = list(out.freqs.values())
 
+    # TODO: Here is the crash! pulses_idx is a list of indexes for
```

Comment:
If it crashes should we have a `try` or `if` to catch exception and return error message?